content/en/docs/Overview/_index.md

```diff
@@ -78,7 +78,7 @@ trouble with the client of your choice.
 ## Road map
 
 This project is in active development. Expect a more polished experience and new features/releases 
-on a frequent basis. Some upcoming features planned: 
+on a frequently. Some upcoming features planned: 
```

Comment:
I think the two are synonymous, no?

Reply:
Yeah but frequent is more appropriate here